services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/enums/RoleKey.java

```diff
@@ -0,0 +1,31 @@
+package com.sourcefuse.jarc.services.usertenantservice.enums;
+
+import com.sourcefuse.jarc.services.usertenantservice.commons.RoleTypes;
+
+public enum RoleKey implements RoleTypes {
+    ADMIN,
```

Comment:
where are the numeric values against the role keys

Reply:
By default ordinals values i.e numeric values are assigned according to the sequence.
but if needed we can assign.